src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_VALUE = 50;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Figure circle = new Circle("White", 10);
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(4)) {
+            case 0: return new Circle((colorSupplier.getRandomColor()).name(),
+                    random.nextInt(MAX_VALUE));
+            case 1: return new RightTriangle((colorSupplier.getRandomColor()).name(),
+                    random.nextInt(MAX_VALUE),random.nextInt(MAX_VALUE));

```

Comment:
```suggestion
            case 1: return new RightTriangle(colorSupplier.getRandomColor().name(),
                    random.nextInt(MAX_VALUE), random.nextInt(MAX_VALUE));
```
Also, let’s use whitespaces between parameters (check it everywhere)